src/Compilers/Core/Portable/Operations/Operation.cs

```diff
@@ -92,6 +106,16 @@ public string Language
 
         public abstract IEnumerable<IOperation> Children { get; }
 
+        SemanticModel IOperation.SemanticModel => _owningSemanticModelOpt?.ContainingModelOrSelf;
+
+        /// <summary>
+        /// Gets the owning semantic model for this operation node.
+        /// Note that this may be different than <see cref="IOperation.SemanticModel"/>, which
+        /// is the semantic model on which <see cref="SemanticModel.GetOperation(SyntaxNode, CancellationToken)"/> was invoked
+        /// to create this node.
+        /// </summary>
+        internal SemanticModel OwningSemanticModel => _owningSemanticModelOpt;
```

Comment:
>OwningSemanticModel [](start = 31, length = 19)

@AlekseyTs I renamed this property to `OwningSemanticModel` to add more clarity and differentiate from publically exposed `IOperation.SemanticModel`